contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -4001,6 +4001,84 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql IMMUTABLE STRICT PARALLEL SAFE;
 
+-- wrapper functions for concat --
+CREATE OR REPLACE FUNCTION sys.concat(VARIADIC args sys.VARCHAR[] DEFAULT '{}')
```

Comment:
We have define the input datatypes as varchar and nvarchar, have we tested with other datatypes like date, datetime, time, smalldatetime, datetime2, datetimeoffset. Will these datatype work?

Reply:
Yes, we have already added tests with all datatypes, and all datatypes other than sql_variant, xml, geometry, geography will work for concat and concat_ws functions.